torch/_dynamo/convert_frame.py

```diff
@@ -2148,10 +2158,14 @@ def __call__(
             or is_skipfile
             or config.disable
             or (
-                should_skip_due_to_torch_dispatch_mode()
-                and not getattr(self._torchdynamo_orig_backend, "_export", False)
+                skip_due_to_dispatch
             )
         ):
+            if getattr(self._torchdynamo_orig_backend, "_one_graph", False) and skip_due_to_dispatch:
+                raise Unsupported(
```

Comment:
Also prefer not using `Unsupported` here - a different exception class would be better